python/graphstorm/dataloading/dataset.py

```diff
@@ -156,6 +186,72 @@ def edge_feat_field(self):
         """the field of edge feature"""
         return self._edge_feat_field
 
+    def load_wg_feat(self, part_config, num_parts, ntype, rank, num_ranks):
+        """Load features from wholegraph memory
+
+        Parameters
+        ----------
+        part_config : str
+            The path of the partition configuration file.
+        num_parts : int
+            The number of partitions of the dataset
+        ntype: str
+            The type of node for which to fetch features.
+        rank: int
+            The rank of the process
+        num_ranks: int
+            The total number of ranks (world size)
+        """
+        import pylibwholegraph.torch as wgth
+        import pylibwholegraph.binding.wholememory_binding as wmb
+
+        if not self._wg_init:
+            class Options:
+                """Set Wholegraph's environment variables """
+            Options.launch_agent = 'pytorch'
+            Options.launch_env_name_world_rank = 'RANK'
+            Options.launch_env_name_world_size = 'WORLD_SIZE'
+            Options.launch_env_name_local_rank = 'LOCAL_RANK'
+            Options.launch_env_name_local_size = 'LOCAL_WORLD_SIZE'
+            Options.launch_env_name_master_addr = 'MASTER_ADDR'
+            Options.launch_env_name_master_port = 'MASTER_PORT'
+            Options.local_rank = rank % role.get_num_trainers()
+            Options.local_size = role.get_num_trainers()
+
+            wgth.distributed_launch(Options, lambda: None)
+            wmb.init(0)
+            th.set_num_threads(1)
```

Comment:
Here, not sure what ` th.set_num_threads(1)` could imply in graphstorm training process? @zheng-da @classicsong

Reply:
BTW, just checked with wholegraph team. It seems `th.set_num_threads(1)` should not be there, and they are fixing it.

Reply:
Lets remove it.

Reply:
Removed